accessibility_scanning/README.md

```diff
@@ -1,3 +1,41 @@
+## Accessibility Scanning using AccessLint
+
+[AccessLintCI](https://github.com/accesslint/accesslint-ci) is an automated accessibility scanning tool. It configured with CircleCI to leave comments on Pull Requests stating potential accessibility problems with the committed code.
+
+### Set up with Jekyll
+
+The set up is detailed [here](https://github.com/accesslint/accesslint-ci#installation) for Rails, but can also easily be configured with Jekyll sites as follows. In the `circle.yml` file of your repo add the following:
```

Comment:
I believe it is 😆 

AccessLint doesn't currently support Travis, but I think it is in their roadmap.